lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -246,16 +264,20 @@ def self.upload(app_id, binary_path)
               UI.message("Uploading the APK/IPA")
               upload_binary(app_id, binary_path)
             end
-            status = upload_status(upload_token, app_id)
+            status, release_id = upload_status(upload_token, app_id)
           end
         end
+        release_id
       end
 
+      # Gets the upload status for the app release
+      #
+      # Returns the status and the id of the release of applicable
       def self.upload_status(app_token, app_id)
         response = connection.get("#{PATH}#{app_id}/upload_status/#{app_token}") do |request|
           request.headers["Authorization"] = "Bearer " + auth_token
         end
-        response.body[:status]
+        return response.body[:status], response.body[:release][:id]
```

Comment:
Piggybacking on an earlier comment about this, I had a similar initial thought that returning tuples isn't ideal...it might not be a far stretch to do something like this:

`return ReleaseUploadStatusResponse.new(response.body)` and then encapsulate examination of the JSON behind that class's interface (an example provided earlier).  That would allow this method not to think about fields on the JSON, and also help us avoid leaking the enums out to the calling code.